Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerOnOffTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// The assembly that this file is part of is still built for the player,
+// even though the assembly itself is marked as a test assembly (this is not
+// expected because test assemblies should not be included in player builds).
+// Because the .NET backend is deprecated in 2018 and removed in 2019 and this
+// issue will likely persist for 2018, this issue is worked around by wrapping all
+// play mode tests in this check.
+
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using System.Collections;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    /// <summary>
+    /// Verify that pointers can be turned on and offvia FocusProvider.SetPointerBehavior
+    /// </summary>
+    public class PointerOnOffTests : BasePlayModeTests
+    {
+        private class PointerStateContainer
+        {
+            public bool? LinePointerEnabled { get; set; }
+            public bool? SpherePointerEnabled { get; set; }
+            public bool? PokePointerEnabled { get; set; }
+            public bool? GazePointerEnabled { get; set; }
```

Comment:
why use bool? when you have PointerBehavior for the three states?

Reply:
because I want to test IsInteractionEnabled of a specific pointer. Otherwise I'm just testing if a setter works. Instead, I'm searching for pointers and actually testing if they are enabled.